source/mqtt/Mqtt5Client.cpp

```diff
@@ -187,6 +187,8 @@ namespace Aws
                 m_socketOptions.SetSocketType(Io::SocketType::Stream);
                 AWS_ZERO_STRUCT(m_packetConnectViewStorage);
                 AWS_ZERO_STRUCT(m_httpProxyOptionsStorage);
+
```

Comment:
Trivial as all heck: extra line break?

Reply:
I'm not sure I want to restart CI for that tbh.